packages/core/src/environment.ts

```diff
@@ -0,0 +1,64 @@
+import { core, devspace } from '@sap/bas-sdk';
+import type { GuidedAnswersQueryFilterOptions, IDE } from '@sap/guided-answers-extension-types';
+
+const devSpaceComponentsMap: { [devspace: string]: string[] } = {
```

Comment:
One open item for which I need your inputs:
- In my view, this mapping should be at a more generic/dedicated place so that in long run, its transparent and easy for others to quickly come and update the devspacetype<->component list.
What do you think? something like https://github.com/SAP/guided-answers-extension/blob/main/packages/ide-extension/src/enhancement/enhancements.json?

Reply:
@theStiggySpd, fully agree, I had it actually in a json, but moved it back to the code because I thought it was too small. 
Anyway, moved it now again to `environment.json` and pushed changes.